de.js

```diff
@@ -0,0 +1,334 @@
+export const en = {
```

Comment:
Didn't see this, I think the first line must be: export const **de** = {

Reply:
Fixed, thanks